test/e2e/conftest.py

```diff
@@ -250,11 +250,17 @@ def pseudo_random_project_name(fixture_request: Any) -> str:
 def pseudo_random_account_name() -> Tuple[str, str]:
     name_max_length = 50
     name_prefix = TEST_ORG_ACCOUNT_NAME_PREFIX
-    random_suffix = str(uuid.uuid4()).replace("-", "")[
-        : name_max_length - len(name_prefix)
-    ]
-    name = f"{name_prefix}{random_suffix}"
-    display_name = f"SDK E2E Test Organization Account - {random_suffix}"
+    random_suffix = str(uuid.uuid4()).replace("-", "")
+    random_suffix_that_fits = random_suffix[: name_max_length - len(name_prefix)]
+    name = f"{name_prefix}{random_suffix_that_fits}"
+    display_name = "SDK E2E Test Organization Account"
+    gh_run_id = os.getenv("GITHUB_RUN_ID")
+    gh_run_attempt = os.getenv("GITHUB_RUN_ATTEMPT")
+    if gh_run_id:
+        display_name += f" - {gh_run_id}:{gh_run_attempt}"
```

Comment:
Can we still add the job name? will be useful to quickly point us to the 3.7 or 3.8 run within the workflow

Reply:
problem is it doesnt add the matrix variable
<img width="792" alt="Screenshot 2022-09-05 at 12 18 25" src="https://user-images.githubusercontent.com/86921811/188428400-4dccaf3c-a2e3-48f3-b9aa-cc00a3c5a929.png">

Reply:
That is annoying. You can pass it as:

```
diff --git a/.github/workflows/check.yml b/.github/workflows/check.yml
index 8e83ef3..908ff64 100644
--- a/.github/workflows/check.yml
+++ b/.github/workflows/check.yml
@@ -95,6 +95,7 @@ jobs:
     env:
       DD_GIT_COMMIT_AUTHOR_EMAIL: ci@layer.ai
       DD_GIT_COMMIT_COMMITTER_EMAIL: ci@layer.ai
+      GITHUB_JOB_PYTHON_VERSION: ${{ matrix.python-version }}
     steps:
       - uses: actions/checkout@v2
       - run: git fetch origin ${{ github.event.pull_request.base.sha }}
diff --git a/.github/workflows/release.yml b/.github/workflows/release.yml
index ff10d97..5431fe2 100644
--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -73,6 +73,7 @@ jobs:
     env:
       DD_GIT_COMMIT_AUTHOR_EMAIL: ci@layer.ai
       DD_GIT_COMMIT_COMMITTER_EMAIL: ci@layer.ai
+      GITHUB_JOB_PYTHON_VERSION: ${{ matrix.python-version }}
     steps:
       - uses: actions/checkout@v2
       - run: git fetch origin ${{ github.event.pull_request.base.sha }}
```

Reply:
this var seems empty for me

<img width="675" alt="Screenshot 2022-09-05 at 13 16 49" src="https://user-images.githubusercontent.com/86921811/188436822-ce323ff0-1767-4a5c-b073-16bb9926a96e.png">